src/vector/backends/awkward.py

```diff
@@ -1597,23 +1597,21 @@ class MomentumRecord4D(MomentumAwkward4D, ak.Record):  # type: ignore[misc]
 
 
 # implementation of behaviors in Numba ########################################
+def lookup_field(record_type: typing.Any, name: str) -> int:
```

Comment:
I did not un-refactor this. Maybe I should. I should at least prepend an underscore because the `awkward` module is public now.